test/choice_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class ChoiceTest < MiniTest::Test
+  def setup
+    @choice = Chiketto::Choice.new 'answer' => {
+        'text' => 'test choice',
+        'html' => '<p>test choice</p>'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.